fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -2244,10 +2244,6 @@ show_param ::=
     {:
         RESULT = new ShowRestoreStmt(db, parser.where);
     :}
-    | KW_BROKER
-    {:
-        RESULT = new ShowBrokerStmt();
-    :}
     | KW_RESOURCES
```

Comment:
I'm not familiar with it, and could you explain why removing this?

Reply:
fine, this statement is defined for old parser and after we use new it's no use in the future, removing it cause there're some ut still use old one in case.